spline-shadow/rest-core/src/main/scala/za/co/absa/spline/gateway/rest/model/OperationDetails.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.gateway.rest.model
+
+import za.co.absa.spline.persistence.{model => persistenceModel}
+
+case class OperationDetails
+(
+  operation: Operation,
+  schemasDefinition: Array[persistenceModel.DataType],
+  schemas: Array[Array[AttributeRef]],
+  inputs: Array[Integer],
```

Comment:
This is to avoid repetitions in the inputs and output schemas. Design it that flatten way give us less information in the webservice results. We treat this flatten structure on the client side to build the tree structure of the schemas.